src/DependencyInjection/Compiler/EntityListenerPass.php

```diff
@@ -25,18 +25,15 @@
  */
 class EntityListenerPass implements CompilerPassInterface
 {
-    use PriorityTaggedServiceTrait;
-
     /** @return void */
     public function process(ContainerBuilder $container)
     {
-        $resolvers = $this->findAndSortTaggedServices('doctrine.orm.entity_listener', $container);
-
         $lazyServiceReferencesByResolver = [];
 
-        foreach ($resolvers as $reference) {
-            $id = $reference->__toString();
-            foreach ($container->getDefinition($id)->getTag('doctrine.orm.entity_listener') as $attributes) {
+        foreach ($container->findTaggedServiceIds('doctrine.orm.entity_listener') as $id => $tags) {
+            usort($tags, static fn (array $a, array $b) => ($a['priority'] ?? 0) <=> ($b['priority'] ?? 0));
```

Comment:
should be
```suggestion
            usort($tags, static fn (array $a, array $b) => ($b['priority'] ?? 0) <=> ($a['priority'] ?? 0));
```

i.e. tags with a larger number as priority come first!?

see [`PriorityTaggedServiceTrait#L113`](https://github.com/symfony/symfony/blob/3fe77f0b5fd9ac22875ded3fa84e950da56d7977/src/Symfony/Component/DependencyInjection/Compiler/PriorityTaggedServiceTrait.php#L113)